CHANGES.rst

```diff
@@ -448,6 +448,9 @@ astropy.time
 - Fixed bug where the location attribute was lost when creating a new ``Time``
   object from an existing ``Time`` or list of ``Time`` objects. [#9969]
 
+- Fixed bug where an error was raised when for a numerical time format, ``val2``
```

Comment:
The grammar in your version is a little difficult.  I'm not sure we need to focus on the format so much as the type of the inputs. Perhaps:
```
- Fixed a bug where an exception occurred when creating a ``Time`` object
  if the ``val1`` argument was a regular double and the ``val2`` argument
  was a ``longdouble``. [#10034]
```

Reply:
Much better, adopted.